src/js/components/recipient/RecipientTooltip.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { throttle } from 'lodash';
+
+import * as Icons from 'components/sharedComponents/icons/Icons';
+
+const propTypes = {
+    content: PropTypes.object, // Pass in a JSX object here as a render prop
+    closeTooltip: PropTypes.func,
+    showInfoTooltip: PropTypes.bool
+};
+
+const tooltipWidth = 160;
+const margin = 15;
+const tooltipPadding = 6;
+
+export default class RecipientTooltip extends React.Component {
```

Comment:
We already have multiple Tooltip components, duplicating the logic required for rendering a tooltip, and we're trying to consolidate them as much as possible. We should be using the `TooltipWrapper` component from `data-transparency-ui`. It also uses a render prop so you should be good for customizing it in that sense.

Reply:
Refactored using `TooltipWrapper`.